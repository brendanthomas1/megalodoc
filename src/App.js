import React, { Component } from 'react';
import logo from './logo.svg';
// NOTE: Using extrnal Bootstrap v4 alpha stylesheet
// import './App.css';

class App extends Component {
  render() {
    return (
      <div className="container-fluid">
        <div className="row">
          <div className="col-4">
            <img src={logo} alt="logo" />
            <h2>Welcome to React</h2>
            <p>To get started, edit <code>src/App.js</code> and save to reload.</p>
          </div>
          <div className="col">
            Main content here
          </div>
        </div>
      </div>
    );
  }
}

export default App;
