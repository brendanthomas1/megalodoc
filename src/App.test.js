import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

it('renders the default view', () => {
  const component = renderer.create(<App></App>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
