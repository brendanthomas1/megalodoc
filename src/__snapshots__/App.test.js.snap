// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the default view 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-4"
    >
      <img
        alt="logo"
        src="test-file-stub"
      />
      <h2>
        Welcome to React
      </h2>
      <p>
        To get started, edit 
        <code>
          src/App.js
        </code>
         and save to reload.
      </p>
    </div>
    <div
      className="col"
    >
      Main content here
    </div>
  </div>
</div>
`;
